Esri/arcgis-maps-sdk-swift-samples
# [New] Monitor changes to map load status

## Description

This PR implements `Monitor changes to map load status` in `Maps` category.
URL to README: https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-MonitorChangesToMapLoadStatus/Shared/Samples/Monitor%20changes%20to%20map%20load%20status

## Linked Issue(s)

- `swift/issues/4175`

## How To Test

* Open the sample to load the map and see the load status displayed on the screen.

## Screenshots

![monitor-changes-to-map-toad-status](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/381b9c4b-a35f-4b57-8102-365d709e15b3)
